docs/docs/aws.md

```diff
@@ -313,6 +313,18 @@ Here are the configurations that users can tune related to this feature:
 | s3.multipart.threshold            | 1.5                                                | the threshold expressed as a factor times the multipart size at which to switch from uploading using a single put object request to uploading using multipart upload  |
 | s3.staging-dir                    | `java.io.tmpdir` property value                    | the directory to hold temporary files  |
 
+### S3 Chunked Encoding
```

Comment:
Why are we introducing a new config to the user? If it's truly necessary, we should add tests to ensure it functions as expected.